gluonnlp/model/transformer.py

```diff
@@ -17,8 +17,8 @@
 # specific language governing permissions and limitations
 # under the License.
 """Encoder and decoder usded in sequence-to-sequence learning."""
-__all__ = ['TransformerEncoder', 'TransformerDecoder',
-           'get_transformer_encoder_decoder', 'get_model']
+__all__ = ['TransformerEncoder', 'PositionwiseFFN', 'TransformerEncoderCell',
+           'transformer_en_de_512']
 
```

Comment:
Originally there was TransformerEncoder. Now there are 3 classes:
BaseTransformerEncoder
-> TransformerEncoder
-> BERTEncoder
where TransformerEncoder and BERTEncoder are just wrappers on top of BaseTransformerEncoder